src/store/cinema/asyncActions.js

```diff
@@ -0,0 +1,20 @@
+import { setCinemaAction, setCinemaErrorAction } from "./slice";
+import formatRequest from "../../helpers/formatRequest";
+
+// eslint-disable-next-line import/prefer-default-export
+export function cinemaFetch(cinemasArray) {
+  if (cinemasArray[0]) {
```

Comment:
`cinemasArray.length` is better to be used instead of checking the first element

Reply:
it's better to use `cinemaList` instead of `cinemasArray`. according to clean code style developers shouldn't use the type of data in the names of variables.